src/app/views/login/SignInController.jsx

```diff
@@ -133,19 +133,21 @@ export class LoginController extends Component {
                 break;
             case "InvalidPassword":
                 errorContents.errorsForBody.push(
-                    <p>
-                        <a href="javascript:document.getElementById('password').focus()" className={"colour--night-shadz"}>
+                    <p key="error-invalid-password">
+                        <a href="javascript:document.getElementById('password').focus()" className="colour--night-shadz">
                             Enter a password
                         </a>
                     </p>
                 );
                 errorContents.passwordMessage = "Enter a password";
                 break;
             case "NotAuthorised":
-                errorContents.errorsForBody.push(<p>Email address or password are incorrect</p>);
+                errorContents.errorsForBody.push(<p key="error-not-authorised">Email address or password are incorrect</p>);
                 break;
             case "TooManyFailedAttempts":
-                errorContents.errorsForBody.push(<p>You've tried to sign in to your account too many times. Please try again later.</p>);
+                errorContents.errorsForBody.push(
```

Comment:
if possible, pls look at my previous comment regarding "capture the error with email/userId"

Reply:
This release doesn't really change the functionality of the SignInController, but I made a few cleanup changes only.
However, the Invalid Email/UserID is handled in the NotAuthorised case. We do not specify which is incorrect as this would be a security risk.